contracts/bonding/BondingManager.sol

```diff
@@ -478,7 +476,8 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
     }
 
     /**
-     * @dev Slash a transcoder. Slashing can be invoked by the protocol or a finder.
+     * @notice Slash a transcoder. Slashing can be invoked by the protocol or a finder.
+     * @dev NOT IN USE - Verifier == address(0)
```

Comment:
This is a detail of the system in production and it could change without re-deploying the BondingManager so let's remove this comment from the source code.

Reply:
fixed in b79bb67